gme/Music_Emu.cpp

```diff
@@ -241,10 +248,22 @@ blargg_err_t Music_Emu::seek( long msec )
 	return seek_samples( msec_to_samples( msec ) );
 }
 
+blargg_err_t Music_Emu::seek_scaled(long msec)
+{
+	require(tempo_ > 0);
```

Comment:
Please, add padding spaces to fit into the project's code style

Reply:
Done. What do you think about adding a `.clang-format` to the project?

Reply:
Sounds a good idea, but not in this PR of course :)